### ISSUE TITLE
I can't save any theme. After closing chrome if I reopen it the theme changes to default

### ISSUE THREAD

**Issue by mayanksakkarwal8** [zombieFox/nightTab#264]
I can't save any theme. After closing chrome if I reopen it the theme changes to default

**Comment by lemonnuggets** [comment]
I'm facing the same issue. I don't have any extensions that screw around with cookies, but just to be safe I still turned off all extensions and tried it again. The problem still persists.
By the way, I use Brave Browser. I'm not sure if that's the cause of this problem, so if anyone could confirm whether they're able to use this extension on Brave that'd be helpful.

**Comment by zombieFox** [comment]
@lemonnuggets Sorry you are having this issue. Unfortunately only Chrome and Firefox is [supported by this project](https://github.com/zombieFox/nightTab/wiki/Browser-support). I can't develop and test in Brave. 

A few others have reported issues with Brave recently. Seems a [restart might fix the issue](https://github.com/zombieFox/nightTab/issues/272) -- but I can't confirm this.

**Comment by phoeniXharsh** [comment]
@zombieFox Yeah the issue is in Brave only. When I customize everything and save my theme, after a couple of hours when I open the new tab, the default blue theme opens up itself. Then I have to close the browser and reload all the tabs to keep the feature working but that is also, for and hour or so. This way it is not feasible to keep your choice of themes.

**Comment by metruzanca** [comment]
@phoeniXharsh @lemonnuggets @mayanksakkarwal8
This is an issue caused by how Brave the privacy focussed browser works. 

NightTab is only officially supported on Chrome and Firefox (see [Browser support](https://github.com/zombieFox/nightTab/wiki/Browser-support) wiki page).
To get it working in Brave, you need to make sure Brave isn't deleting site data.

From what I could tell, Brave does this in two places:

![image](https://user-images.githubusercontent.com/32832892/135937374-f30818b1-3da9-4d30-ba0f-73c6aa98c6a3.png)
![image](https://user-images.githubusercontent.com/32832892/135937383-7b13e278-8148-4f05-bba3-295ae116ca1e.png)

If you search "data" in the search bar, you should find these two sections easily.
